XCUITests/FxScreenGraph.swift

```diff
@@ -483,6 +486,15 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.press(app.tables["History List"].cells.element(boundBy: 2), to: HistoryPanelContextMenu)
         screenState.tap(app.cells["HistoryPanel.recentlyClosedCell"], to: HistoryRecentlyClosed)
         screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
+        screenState.gesture(forAction: Action.ClearLastHourHistory) { userState in
+            app.tables["History List"].staticTexts["Clear Recent History…"].tap()
+            app.sheets["Clearing Recent History will remove history, cookies, and other browser data."].buttons["The Last Hour"].tap()
+        }
+        screenState.gesture(forAction: Action.ClearTodayHistory) { userState in
```

Comment:
This action is pretty similar to the previous one, there is a lot of code duplication...Why don't we create an action to show the clear recent history menu and then in the test a function to select the option...for example...
```
private func tapOnClearRecentHistoryOption(optionSelected: String) {
        app.sheets.buttons[optionSelected].tap()
    }
```
and the test would be something like...
```
  navigator.performAction(Action.ClearLastHourHistory)
  tapOnClearRecentHistoryOption(optionSelected: "Today")
```
Does this make sense?